src/models/factories/wistiaVideoFieldFactory.ts

```diff
@@ -0,0 +1,24 @@
+import { DeepPartial } from "@/types/utils/DeepPartial";
+import { ExpandedFieldDetails } from "../types/ExpandedFieldDetails";
+import merge from "lodash/merge";
+
+export const wistiaVideoFieldFactory = (
+  props: {
+    id: string;
+    name: string;
+  } & DeepPartial<Omit<ExpandedFieldDetails, "id" | "name" | "type">>,
+): ExpandedFieldDetails =>
+  merge(
+    {
+      type: "Object",
+      localized: false,
+      validations: [],
+      disabled: false,
+      omitted: false,
+      editorInterface: {
+        widgetId: "6StWOM1AZBDHDjynDkm1iz",
+        widgetNamespace: "app",
+      },
+    } as Omit<ExpandedFieldDetails, "id" | "name">,
```

Comment:
you dont need all the options there, only if you want to intentional force a value, but for false values


```suggestion
    {
      type: "Object",
      localized: false,
      editorInterface: {
        widgetId: "6StWOM1AZBDHDjynDkm1iz",
        widgetNamespace: "app",
      },
    } as Omit<ExpandedFieldDetails, "id" | "name">,
```